<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>259</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>282</width>
    <height>259</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>282</width>
    <height>259</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Rubrica</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icone.qrc">
    <normaloff>:/Immagini/book.ico</normaloff>:/Immagini/book.ico</iconset>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>259</width>
     <height>205</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>259</width>
     <height>205</height>
    </size>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dati Anagrafici</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>247</width>
      <height>163</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="Nome">
       <property name="text">
        <string>Nome(*)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nomeEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Cognome">
       <property name="text">
        <string>Cognome</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="cognomeEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="Data_di_Nascita">
       <property name="text">
        <string>Data di Nascita</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="Provincia">
       <property name="text">
        <string>Provincia di Nascita</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="provinciaBox"/>
     </item>
     <item row="6" column="0">
      <layout class="QGridLayout" name="gridLayout"/>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="residenzaBox"/>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="Residenza">
       <property name="text">
        <string>Residenza</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>271</width>
      <height>155</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="selectedDate">
     <date>
      <year>2000</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>calendarWidget</zorder>
   <zorder>label</zorder>
   <zorder>formLayoutWidget</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionRead"/>
   <addaction name="separator"/>
   <addaction name="actionInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/Immagini/file-128.png</normaloff>:/Immagini/file-128.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/Immagini/External_Drive-Black.png</normaloff>:/Immagini/External_Drive-Black.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/Immagini/save-128.png</normaloff>:/Immagini/save-128.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/Immagini/symbol_information.png</normaloff>:/Immagini/symbol_information.png</iconset>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionRead">
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/Immagini/Read-128.png</normaloff>:/Immagini/Read-128.png</iconset>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>nomeEdit</tabstop>
  <tabstop>cognomeEdit</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>provinciaBox</tabstop>
  <tabstop>residenzaBox</tabstop>
  <tabstop>calendarWidget</tabstop>
 </tabstops>
 <resources>
  <include location="icone.qrc"/>
 </resources>
 <connections/>
</ui>
